src/schema/schema.graphql

```diff
@@ -280,10 +280,36 @@ type AuthResponse {
 	skin: [Skin]
 }
 
+type ScratchCode{
+	ScratchCode: String
+}
+
+type EnableTwoFactorAuthResponse {
+	secret: [Secret]
+	scratchCodes: [ScratchCode]
```

Comment:
this is mandatory property

Reply:
mandatory property cannot be set to a type

Reply:
done